### ISSUE TITLE
It never finishes - or I just cant see the results

### ISSUE THREAD

**Issue by sztojkatamas** [LazarZsolt/LoadTest#1]
docker run loadtest python /LoadTest.py -m 5 -l 1  -t 100000 http://www.sg.hu
Run1
Initializing 1 workers
Waiting for workers to finish
Run2
Initializing 5 workers
Waiting for workers to finish

**Comment by LazarZsolt** [comment]
First of all sorry for the late answer. I was not notified.

In your case: 
The test first runned with one worker (-l parameter), then it runned with 5 workers.
By default the numer of workers is increased by 5, in every round. 
In the second run the number of workers reached the maximum (given with -m),
So the test run finished and the result you see here is the end result.
The last parameter you gave (-t) was how long the worker waits for the response from the server.

So in short. In your case the test runned and finished, and the result you see is the endresult.

The test stops if one of these cases:
1.  The number of workers reaches the maximum (this was happening here)
2.  In the last run one or more workers fail to connect
3.  On KeyboardInterrupt


I hope my answer was usable. If you have any questions feel free to ask.
